images/stable-baselines/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG on
+FROM python:${on}
+
+RUN apt-get -qy update && \
+    apt-get -qy install libgl1 && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get -qy clean
```

Comment:
There are other libs that need to be installed to use SB: https://stable-baselines.readthedocs.io/en/master/guide/install.html

Reply:
It's a bit unclear to me why these would be needed. They are all build time dependencies..

Reply:
Mmm the docs are confusing, probably it is only needed to build it from sources? I d say to leave it as it is and I will test it with my agent....